Keas.Mvc/Controllers/Api/KeysController.cs

```diff
@@ -42,10 +42,9 @@ select new
                 key = key,
                 id = key.Id,
                 code = key.Code,
-                serialsTotalCount = 
-                    (key.Serials).Count(),
+                serialsTotalCount = (from s in key.Serials where s.Status == "Active" select s).Count(),
                 serialsInUseCount = 
-                    (from s in key.Serials where s.KeySerialAssignment != null select s ).Count(),
+                    (from s in key.Serials where s.KeySerialAssignment != null && s.Status == "Active" select s ).Count(),
```

Comment:
@jcubbage i'm not sure this one is worth doing.  We shouldn't have a non-active serial assigned to someone.  And if we do, it should be in-use.  I know right now we do allow that... but I'd say remove this and we'll fix the problem on the other end?  Alternately we can keep this in to be safe, though I think it's just extra db work we don't need

Reply:
I'd say keep this way until the other way is fixed.